packages/storybook/stories/QuickStarts/snippets/CallAppStateful.snippet.tsx

```diff
@@ -0,0 +1,46 @@
+import { AzureCommunicationTokenCredential } from '@azure/communication-common';
+import { DEFAULT_COMPONENT_ICONS, createStatefulCallClient, StatefulCallClient } from '@azure/communication-react';
+import React, { useEffect, useState } from 'react';
+import { registerIcons } from '@fluentui/react';
+import { Call, CallAgent } from '@azure/communication-calling';
+
+function App(): JSX.Element {
+  registerIcons({ icons: DEFAULT_COMPONENT_ICONS });
+
+  const userAccessToken = '<Azure Communication Services Resource Access Token>';
+  const userId = '<User Id associated to the token>';
+  const tokenCredential = new AzureCommunicationTokenCredential(userAccessToken);
+  const groupId = '<Generated GUID groupd id>';
+  const displayName = '<Display Name>';
+
+  const [statefulCallClient, setStatefulCallClient] = useState<StatefulCallClient>();
+  const [callAgent, setCallAgent] = useState<CallAgent>();
+  const [call, setCall] = useState<Call>();
+
+  useEffect(() => {
+    setStatefulCallClient(
+      createStatefulCallClient({
+        userId: { communicationUserId: userId }
```

Comment:
Consider using fromFlatCommunicationIdentifier in all quickstarts. Raw manipulation of user IDs is dangerous.